apps/shipments/models/shipment.py

```diff
@@ -511,6 +523,24 @@ def drop_off(self, document_id=None, raw_asset_physical_id=None, **kwargs):
 
         self.delivery_act = datetime.now(timezone.utc)  # TODO: pull from action parameters?
 
+    def validate_gtx(self):
```

Comment:
We want to call this for any action where we accept the `asset_physical_id` to validate that the id tag on the asset is valid per GTX.  This would include the `drop_off` action.

Reply:
The reason I didn't include the check on the `drop_off` was because the `asset_physical_id` wasn't changing, just validating against the `asset_physical_id` on the object itself.